modules/configuration/help/project.md

```diff
@@ -0,0 +1,8 @@
+# Project
+
+Use this page to manage the configuration of existing projects, or to add a new one. Your projects are listed on the left side of the screen. Click on any project name to edits its settings, and click **Save**. 
+
+Click **New ProjectID** from the left side to crate a new project. Populate the fields and click **Save**. Your new project should now appear in the list, after you refresh the page. 
+
+
+Define all projectID-subprojectID relationships by populating the project_rel table.
```

Comment:
I think this sentence in the help text is going to confuse users.. how do they populate the table? Do they need a dev to do it?

Reply:
I agree, these lines can probably be deleted. It's more appropriate to the module README or a wiki page.

Reply:
deleted